3_Inference/Detector.py

```diff
@@ -293,5 +294,13 @@ def get_parent_dir(n=1):
                 len(input_video_paths), end - start
             )
         )
+    # for Webcam uncomment the below code and the one in 2_Training/src/keras_yolo3/yolo.py
+    """
+    start = timer()
+    detect_video(yolo)
+
+    end = timer()
+    print("Processed {} videos in {:.1f}sec".format( len(input_video_paths), end - start))
+    """
```

Comment:
Instead of uncommenting, make it an (optional) CLI argument

Reply:
Ok i' ll to do all these changes ASAP